README.md

```diff
@@ -36,7 +36,14 @@ steps:
       # If not specified semantic-release will use its default branches
       # configuration, as specified in their Javascript API documentation:
       # https://github.com/semantic-release/semantic-release/blob/master/docs/usage/configuration.md#branches
-      release-branch: 'feat/my-feature-branch'
+      release-branches: [
+        '+([0-9])?(.{+([0-9]),x}).x',
+        'master',
+        'next',
+        'next-major',
+        { name: 'beta', prerelease: true },
+        { name: 'alpha', prerelease: true },
+      ],
```

Comment:
My hunch, and it's just a hunch, is that this won't really translate into a JSON string when ingested by `getInput`. Did you get a chance to test this in CI?